index.html

```diff
@@ -0,0 +1,174 @@
+                    <p class="text-light fs-16">Wed, Jan 20, 2021</p>
+                </div>
+            </div>
+
+            <div class="col-6 col-lg-4  item-height d-flex align-items-center justify-content-start justify-content-lg-center">
+                <div>
+                    <img src="logo.svg" alt="" class="logo">
+                </div>
+            </div>
+
+            <div class="col-6 col-lg-4 item-height d-flex align-items-center justify-content-end">
+                <i class="fa fa-user text-light d-md-none"></i>
+
+                <div class="d-flex">
+                    <button type="button" class="btn d-none d-md-block  text-light fs-18"><strong>LOGIN</strong></button>
+                    <button type="button" class="btn btn-primary btn-sub text-dark"><strong>SUBSCRIBE<span class="heavy font-roboto">></span></strong></button>
+                </div>
+
+                <i class="fa fa-bars text-light d-lg-none"></i>
+            </div>
+        </div>
+
+        <nav class="d-none d-lg-block">
+            <div  class="nav-links">
+                <ul  class="d-flex align-items-center list-unstyled link-list font-roboto border-bottom">
+                    <li class="border-end">
+                        <a href="#" class="text-dark">U.S.</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">World</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Business</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Tech & Science</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Culture</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Newsgeek</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Sports</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Health</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">The Debate</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Experts</a>
+                    </li>
+                    <li class="border-end">
+                        <a class="text-dark" href="#">Vantage</a>
+                    </li>
+                    <li>
+                        <div>
+                            <input type="text" id="search" name="search" class="search font-roboto" placeholder="Search">
+                            <label for="search"><i class="fa fa-search"></i></label>
```

Comment:
The search input must be wrapped in the form tag.

Reply:
Got it